src/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import heroimage from '../../../assets/hero/heroImage.png'
+import styles from "../../styles/Hero.module.css";
+
+export const Hero:React.FC = () => {
+  return (
+    <section className={styles.container}>
+      <div className={styles.content}>
+        <h1 className={styles.title}>Hi, I'm Sakshi</h1>
+        <p className={styles.description}>
+          I'm a Frontend developer.
+          Aspiring software engineer at Simform <br>
+          </br>
+          with hands-on training in coding, web development.

```

Comment:
We can have a single `br`. 
```suggestion
          Aspiring software engineer at Simform
          <br />
          with hands-on training in coding, web development.

```

Reply:
fixed https://github.com/Sakshi-Simform/portfolio/pull/2/commits/4ae9a36bf9210ac7d234f2cdd473b51d7f5d327f